graphics_altivec.cpp

```diff
@@ -47,13 +47,13 @@ void imageFilterMean_Altivec(unsigned char *src1, unsigned char *src2, unsigned
     }
 
     // Do bulk of processing using Altivec (find the mean of 16 8-bit unsigned integers, with saturation)
-    vector unsigned char rshft = vec_splat_u8(0x1);
+    __vector unsigned char rshft = vec_splat_u8(0x1);
```

Comment:
The non-prefixed `vector` declaration was Motorola syntax many moons ago -- it has a different meaning in modern C++ compilers. `__vector` is IBM and also GNU syntax for VMX/AltiVec vector intrinsics.